src/pheval/config_parser.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+import yaml
+from serde import serde
+from serde.yaml import from_yaml
+
+
+@serde
+@dataclass
+class InputDirConfig:
+    tool: str
+    software_directory: Path or None
+    software_config: Path or None
+    input_directory: Path
+    phenotype_only: bool
```

Comment:
This seems out of place here. Can we add some documentation what each of these parameters exactly are?

Reply:
(maybe this is not out of place, just need to understand why it is here)

Reply:
added some documentation following how @souzadevinicius has structured his docstrings